src/library/scala/collection/LinearSeqOptimized.scala

```diff
@@ -117,20 +117,20 @@ trait LinearSeqOptimized[+A, +Repr <: LinearSeqOptimized[A, Repr]] extends Linea
   }
 
   override /*TraversableLike*/
-  def foldLeft[B](z: B)(f: (B, A) => B): B = {
+  def foldLeft[B](z: B)(@deprecatedName('f) op: (B, A) => B): B = {
```

Comment:
Also aligned in a private object in `RedBlackTree.scala`. The access to this object is `private` so voided the need for `deprecatedName`.